src/com/owncloud/android/files/InstantUploadBroadcastReceiver.java

```diff
@@ -122,6 +123,25 @@ private void handleNewPictureAction(Context context, Intent intent) {
         i.putExtra(FileUploader.KEY_UPLOAD_TYPE, FileUploader.UPLOAD_SINGLE_FILE);
         i.putExtra(FileUploader.KEY_MIME_TYPE, mime_type);
         i.putExtra(FileUploader.KEY_INSTANT_UPLOAD, true);
+
+        // instant upload behaviour
+        SharedPreferences appPreferences = PreferenceManager.getDefaultSharedPreferences(context);
```

Comment:
This block is three times in the same file. 
Could you include this code in a method and use it instead of repeat it?